wolfssl/wolfcrypt/asn.h

```diff
@@ -2630,6 +2632,8 @@ WOLFSSL_LOCAL int    InitOcspRequest(OcspRequest* req, DecodedCert* cert,
 WOLFSSL_LOCAL void   FreeOcspRequest(OcspRequest* req);
 WOLFSSL_LOCAL int    EncodeOcspRequest(OcspRequest* req, byte* output,
                                        word32 size);
+WOLFSSL_LOCAL int CreateOcspRequest(struct WOLFSSL* ssl, OcspRequest* request,
```

Comment:
This function declaration should not be in wolfCrypt... please move to internal.h and delete `struct WOLFSSL;`